web/src/message_scroll.js

```diff
@@ -154,12 +154,21 @@ export function initialize() {
 
         if (event.mark_read && event.previously_selected_id !== -1) {
             // Mark messages between old pointer and new pointer as read
-            let messages;
             if (event.id < event.previously_selected_id) {
-                messages = event.msg_list.message_range(event.id, event.previously_selected_id);
+                // We don't mark messages as read when the pointer moves up.
+                return;
+            }
```

Comment:
Why is it desirable to remove the previous behavior of having scrolling up mark things as read while crossing it? I don't see any clear discussion of that.

Reply:
I was trying to implement "messages get marked as read when you move the blue box past them" otherwise if you move the blue box up while on the first unread message, it will mark it as read which looks weird.

Reply:
Yeah, maybe it's good to make you move up past them and then you can mark moving down anyway.